spec/support/factory_girl.rb

```diff
@@ -7,4 +7,15 @@
     password_confirmation 'password'
   end
 
+  factory :job do
+    title 'software engineer'
+    company 'google'
+    city  'palo alto'
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.